tests/main/main.test.js

```diff
@@ -115,7 +106,7 @@ describe('extractZipInplace', () => {
   let file2Path;
   let doneZipping = false;
 
-  beforeEach(() => {
+  beforeEach((done) => {
```

Comment:
Could you explain what's going on with `done` here? I couldn't find anything about it in the jest docs

Reply:
Here's the docs: https://jestjs.io/docs/asynchronous

And then from https://jestjs.io/docs/setup-teardown:
> beforeEach and afterEach can handle asynchronous code in the same ways that tests can handle asynchronous code - they can either take a done parameter or return a promise.

In this case there is async stuff happening in the setup; we need to wait for the `on('close'...` handler to finish. But there is no Promise we're waiting for, so `done` seemed like a way to control things.

Reply:
thanks!